src/pipeline_config.py

```diff
@@ -104,16 +95,18 @@
                                   'epoch_every': 1},
             'validation_monitor': {
                 'epoch_every': 1,
-                'data_dir': params.train_imgs_dir,
-                'validate_with_map': params.validate_with_map,
-                'small_annotations_size': params.small_annotations_size,
+                # 'data_dir': params.train_imgs_dir,
```

Comment:
@taraspiotr Should this be commented out?

Reply:
@jakubczakon yes, until we'll implement custom callbacks, those parameters aren't suitable for the ones from toolkit